### DISCUSSION TITLE
How do I skip using `_quarto.yml` for a specific document in the folder?

### DISCUSSION THREAD

**Discussion by royfrancis** [quarto-dev/quarto-cli#5224] [Q&A] [answered]
In a website project context, the settings in **_quarto.yml** is used during `quarto render document.qmd` by default. How do I skip using **_quarto.yml** for a specific document in the folder?

I don't see anything obvious in `quarto render --help`, but I am thinking something like `quarto render --disable-config document.qmd` which should only use the yaml metadata specified within that document.

**Comment by mcanouil** [comment]
`_quarto.yml` is required for Quarto projects such as book and website, so for books and website you have to use it.

I assume, you want to render a document that is part of a project, but you want to render it as a standalone document (assuming it has the proper yaml header), am I correct?

**Reply by royfrancis** [reply]
yes exactly! I have one document which I would like to show the rendered output as "default" output for teaching purposes. It should not take any of the settings or style from **_quarto.yml**.

**Comment by cderv** [comment, marked as answer]
I think you should leverage the different project feature regarding how config and metadata are merged so that the document you want to render has only the option you need. 

* You can split the metadata in subfolder, and inside document and it will be merged with `_quarto.yml` https://quarto.org/docs/projects/quarto-projects.html#shared-metadata
If you have different part of your project that needs to have different global config, you should make your `_quarto.yml` minimal, and use the `_metadata.yml` or YAML in-document header to pass the configuration you need. 

* You can also [Project Profile](https://quarto.org/docs/projects/profiles.html) to render some document with a specific profile and not the default one. 

Hope it helps

**Reply by royfrancis** [reply]
Thanks! I was aware of the shared metadata. The downside is that I will have to make my `_quarto.yml` minimal and have child yml in subfolders. It's a lot of effort and reorganisation just to disable styles for one document.
I wasn't aware of project profiles, but I will look into that.

**Reply by cderv** [reply]
> It's a lot of effort and reorganisation just to disable styles for one document.

If you can share a demo project of what you want to do, maybe we'll be able to suggest a better solution. 

Usually when you have a document you want to include in Quarto project like a website, but you need to be an independant document, you can ignore this output from the render target. https://quarto.org/docs/projects/quarto-projects.html#render-targets 
Then you can have [Project Scripts](https://quarto.org/docs/projects/scripts.html) to render those file externally from the all project rendering as they are not supposed to follow the site rules (IIUC). [Site resource](https://quarto.org/docs/websites/website-tools.html#site-resources) can then be configured to include those content in the output website for download or anything else. 

Also, depending on what you mean by styling, you should be able to override the `format:` part of the `_quarto.yml` from within your sub folder `_metadata.yaml` or your in document YAML header. If you use other style, it should replace the default one. 

Overall it depends if you need site integration (navbar, side bar...) or if those documents are completely on their own. 

Example with the Datascience in a box website where the all `_slide` folder is handle differently, and included as external resource only
https://github.com/rstudio-education/datascience-box/blob/5620561ea2ee962fd7b710d273f39efc37ce99e0/_quarto.yml#L5

**Reply by royfrancis** [reply]
Thanks for the suggestions and examples. Very helpful! I will play around with this a bit more. And get back if needed.
